shared/src/prompts.rs

```diff
@@ -565,6 +565,14 @@ pub fn ask_endpoint(listen_port: u16) -> Result<Endpoint, Error> {
         .interact()?
     {
         publicip::get_any(Preference::Ipv4)
+    } else if Confirm::with_theme(&*THEME)
+        .wait_for_newline(true)
+        .with_prompt(
+            "Use an unspecified IP address? (this can occur if you do not have a fixed global IP)",
+        )
+        .interact()?
+    {
+        Some(IpAddr::V6(Ipv6Addr::UNSPECIFIED))
```

Comment:
I guess it does not really make a difference whether the unspecified IP address is IPv4 or IPv6? Still, we generally default to IPv4, so we may as well follow suit here.